### ISSUE TITLE
Don't see the spaces-renamer version of mission control bar

### ISSUE THREAD

**Issue by bomtailey** [dado3212/spaces-renamer#76]
Believe I have followed all the prescribed steps for installing spaces-renamer but if I invoke mission control I only ever see the standard Mac version (gray bar with Mac names: Desktop 1 etc.). What am I missing?

**Versions**
1. MySIMBL version - using MacForge 1.1.0
2. Spaces Renamer version - 1.9.1
3. Mac OS version 11.5

**Comment by kathryntsai** [comment]
Hi!  I'm having a similar issue with the same Spaces Renamer and MacForge versions, but I can open another one if this is different -- I'm running Big Sur 11.6 on a Macbook Pro with 2.3 GHz 8-Core Intel Core i9 processor.

I've followed the tutorial -- downloaded MacForge, disabled SIP, downloaded the .zip file, opened the bundle/plugin in MacForge where it appears, double-clicked the Application to move it to the /Applications folder (it shows up in Login items). The menu bar icon appears and I've been able to type new names and click "Update Names," but the Desktop names still rename the same.

I tried deleting everything (the bundle from MacForge, the application) and restarting everything, but (a) that doesn't work, and (b) the names I had used in my previous attempt are still there.

What might I be doing wrong? Thank you.

**Comment by dumblob** [comment]
It's a bit silly to post this under nearly all open issues in this tracker but here we go.

I had a similar problem and the following sentence I quote (from https://github.com/dado3212/spaces-renamer/issues/74#issuecomment-818411128 ) explained what was the reason (it seems the MacForge as well as spaces-renamer were incorrectly installed for an unknown reason - because in `macOS -> Preferences -> Users & Groups > Login Items` I had only **yellow triangles** for both entries (MacForge as well as spaces-renamer) which means the file the entries were pointing to didn't exist or weren't readable for some reason (permissions?). So I've removed both entries with the ` - ` button (otherwise they could interfere under certain conditions with the repeated manual reinstallation as discussed below).

>If you do not see spaces-renamer in your MacForge plugins then updating names in the spaces-renamer application in your top drawer will not work.

So I followed the recommendation to **not** install spaces-renamer through the "recommendations" in MacForge window but **manually** downloaded space-renamer and meticulously followed the steps in the readme in this repo - i.e. first drag & drop `*.bundle` on a running MacForge window, then `killall -9 Dock`, then first double click the `SpacesRenamer.app` file. Then the records in `macOS -> Preferences -> Users & Groups > Login Items` were without yellow triangles and fully working.

And then do **not** be afraid by the button `Update Names` being **greyed out** but just press it anyway!

Feel free to close this issue if it works now for you!
